ballerina/build.gradle

```diff
@@ -157,6 +165,11 @@ task packageDist(type: Zip) {
         from "build/target/extracted-distributions/awslambda-extension-balo-zip/awslambda-artifacts/awslambda"
     }
 
+    /* Standard Libraries */
+    into("${parentDir}/bir-cache/ballerina/socket") {
```

Comment:
Can we make this read from configuration. I mean make common task for copying native and ballerina module jars for each std lib. If not we need duplicate copy commands to each std lib.

Reply:
We need to do for other places as well.  ex. copy bir cache, copy bre/lib etc

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-distribution/pull/482/commits/6d4834b4d7ebd1d951d0a200ebf142766dee951d